.github/workflows/ci.yaml

```diff
@@ -89,6 +89,18 @@ jobs:
         name: pgt-linux-x86-64
         path: /home/runner/.local/bin/pgt
         if-no-files-found: error
+    - name: Upload latest pgt release
+      if: ${{ matrix.project == 'pgt' && github.ref != 'refs/heads/main' }}
```

Comment:
@dkubb Shouldn't  `github.ref != 'refs/heads/main'` be `github.ref == 'refs/heads/main'` for it to run on only `main` branch builds ?

Reply:
@epicallan Good catch! You are correct. Fixed.